__tests__/happy.uploadAttachments.js

```diff
@@ -87,17 +89,25 @@ describe('Happy path', () => {
               ])
             ]);
 
-            expect(attachments[0]).not.toBe(attachments[1]);
-            expect(~actuals.indexOf(expecteds[0])).toBeTruthy();
-            expect(~actuals.indexOf(expecteds[1])).toBeTruthy();
+            const actualThumbnailUrls = attachments.map(({ thumbnailUrl }) => thumbnailUrl);
+
+            return (
+              attachments[0] !== attachments[1]
+              && actualContents.includes(expectedContents[0])
+              && actualContents.includes(expectedContents[1])
+              && actualThumbnailUrls.includes(activityFromUser.attachments[0].thumbnailUrl)
+              && actualThumbnailUrls.includes(activityFromUser.attachments[1].thumbnailUrl)
+            );
 
             // Use the commented code below after bug #194 is fixed.
             // https://github.com/microsoft/BotFramework-DirectLineJS/issues/194
 
-            // await expect(fetchAsBase64(attachments[0].contentUrl)).resolves.toBe(await fetchAsBase64(activityFromUser.attachments[0].contentUrl));
-            // await expect(fetchAsBase64(attachments[1].contentUrl)).resolves.toBe(await fetchAsBase64(activityFromUser.attachments[1].contentUrl));
-
-            return true;
+            // return (
```

Comment:
Do we want to keep this commented out?

Reply:
Yap, keep commenting out so we don't need to think how to rewrite that code, later.